### ISSUE TITLE
GoogleTranslateOption - cancel editing messages = blank message

### ISSUE THREAD

**Issue by ToxicMethASS** [mwittrien/BetterDiscordAddons#264]
i wrote a message in a different language (specifically my own, doesn't matter however) and i decided to translate it back to english to see how it translates. 
while it was in the whole "translated" mode, i decided to press the up arrow on my keyboard to see what happens if i go edit the message; will the source stay in my language or did it carry the translation over to the edit box (note: the same can be replicated with right clicking and pressing "Edit Message"). to my expectation it stayed in source (not saying it should be changed), so i close out of the edit message box with esc (same can be done with the cancel in the corner). 

what happens next: the message becomes completely blank and appears to be missing margins on the bottom

tl;dr how to replicate

1. send a message
2. translate the message with GTO
3. open the edit message area for the message
4. close the edit message box without editing
5. the message should be blank

this does not stay after switching channels.

**Comment by mwittrien** [comment]
Will be fixed with the recent light time update fix tomorrow
